### ISSUE TITLE
Applying dropout at inference time

### ISSUE THREAD

**Issue by ine1113** [NVIDIA/tacotron2#26]
Hello,

Fantastic implementation, and thank you for open-sourcing it!

I've noticed your implementation applies dropouts during inference time to several different layers, but to my understanding, dropouts should only be applied to the decoder prenet layers at inference time. I was wondering if there was any reasoning behind the current implementation.

Thanks!

**Comment by rafaelvalle** [comment]
In our implementation, during inference time we disable dropout, for all layers except the prenet layer, by calling `model.eval`. Note that calling `model.eval()` has no effect on the prenet's dropout because `training` is hardcoded to true.
https://github.com/NVIDIA/tacotron2/blob/master/model.py#L99

**Comment by ine1113** [comment]
Understood. Thanks!
